### ISSUE TITLE
DNS entry 255.255.255.255 crashes aquatone-scan

### ISSUE THREAD

**Issue by LukasReschke** [michenriksen/aquatone#11]
Example:

```
255.255.255.255 ping.foo.com
```

```
An unexpected error occurred: Errno::EACCES: Permission denied - connect(2) for "255.255.255.255" port 80
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/commands/scan.rb:85:in `initialize'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/commands/scan.rb:85:in `new'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/commands/scan.rb:85:in `block in port_open?'
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/timeout.rb:93:in `block in timeout'
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/timeout.rb:33:in `block in catch'
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/timeout.rb:33:in `catch'
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/timeout.rb:33:in `catch'
/usr/local/Cellar/ruby/2.4.1_1/lib/ruby/2.4.0/timeout.rb:108:in `timeout'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/commands/scan.rb:84:in `port_open?'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/commands/scan.rb:53:in `block (2 levels) in scan_ports'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/thread_pool.rb:13:in `block (4 levels) in initialize'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/thread_pool.rb:11:in `loop'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/thread_pool.rb:11:in `block (3 levels) in initialize'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/thread_pool.rb:10:in `catch'
/usr/local/lib/ruby/gems/2.4.0/gems/aquatone-0.1.1/lib/aquatone/thread_pool.rb:10:in `block (2 levels) in initialize'
```

**Comment by michenriksen** [comment]
Hehe, interesting, I will look into ignoring this IP. :)

Thanks for reporting!

**Comment by michenriksen** [comment]
This issue is fixed in version 0.2.0.
